code/modules/mob/living/simple_mob/subtypes/animal/alien animals/spacewhale.dm

```diff
@@ -0,0 +1,203 @@
+/datum/category_item/catalogue/fauna/spacewhale
+	name = "Alien Wildlife - Star Treader"
```

Comment:
~~I can't tell if its a placeholder or a forgotten copypaste but either way will just point it out.~~